src/registrar/templates/domain_dnssec.html

```diff
@@ -27,7 +27,7 @@
   {% endif %}
   {% endblock breadcrumb %}
 
-  <h1>DNSSEC</h1>
+  <h1 id="domain-dnssec">DNSSEC</h1>
```

Comment:
Since this id is used to init the JS, why not use the 'other' unique id you introduced on the new form: `disable-dnssec-form`

Reply:
I think I will leave the ids separate. They are used for different purposes. If one of the forms changes, I don't want it to inadvertently remove the js initiation.